README.md

```diff
@@ -55,6 +55,33 @@ You'll need [Clang 3.7](http://llvm.org/releases/download.html#3.7.0).
 You can then pass the contents of `prog.o` to `ubpf_load_elf`, or to the stdin of
 the `vm/test` binary.
 
+## Benchmarking and Instruction Counts
+
+Using the ```-i``` options instructs the test program to count the
+number of eBPF and native instructions executed when implementing the
+program. In the case of a JITed progam the native instructions only
+are recorded but a record of the number of instructions needed to
+generated the JITed code is added. Note that this requires appropriate
+privilidges in order to be run.
+
+A number of simple benchmark programs are included in the bench
+folder. Compile them to eBPF in the usual way and include a memory
+file to provide the input data. Note the memory files are not included
+in the repo but an example of how to generate only and apply it to a
+benchmark is given below.
+```
+dd if=/dev/urandom of=./bench/crc32.mem bs=4k count=2
+clang -O2 -target bpf -c ./bench/crc32.c -o ./bench/crc32.o
+sudo ./vm/test -i -j -m ./bench/crc32.mem ./bench/crc32.o 
```

Comment:
If the example was executed with `-j`, then shouldn't we get instruction count for cmpl != 0 and for vm == 0?